internal/control-panel/registry.lua

```diff
@@ -95,6 +95,8 @@ COMMANDS_BY_IDX = {
     {command='preserve-tombs', group='bugfix', mode='enable', default=true},
     {command='reaction-gloves', help_command='tweak', group='bugfix', mode='tweak', default=true,
         desc='Fixes reactions not producing gloves in sets with correct handedness.'},
+    {command='fix/shooting', group='bugfix', mode='tweak', default=true,
```

Comment:
this is neither a command nor a tweak. you can remove it from the control-panel registry entirely since it will be controlled by the overlay framework